lib/ui/page/home/page/my_profile/delete_email/controller.dart

```diff
@@ -37,21 +37,19 @@ class DeleteEmailController extends GetxController {
     this._myUserService,
     this._authService, {
     required this.email,
-    this.pop,
   });
 
   /// [UserEmail] the [DeleteEmailView] is about.
   final UserEmail email;
 
-  /// Callback, called when a [DeleteEmailView] this controller is bound to
-  /// should be popped from the [Navigator].
-  final void Function()? pop;
+  /// Indicator whether [UserEmail] has been deleted successfully.
+  final RxBool deleted = RxBool(false);
```

Comment:
Shouldn't we use the same approach as in `AddEmail` modal? Code should follow the same solutions ideally so that it's easier to understand and read, don't you agree? If you take a look at different modals, you'll find there's a `switch(c.someEnum.value) { }` approach is often used.